src/vos/vos_iterator.c

```diff
@@ -609,6 +665,27 @@ vos_iter_cb(vos_iter_cb_t iter_cb, daos_handle_t ih, vos_iter_entry_t *iter_ent,
 	return rc;
 }
 
+#define JUMP_TO_STAGE(rc, next_label, probe_label, abort_label)				\
```

Comment:
(style)  Macros with flow control statements should be avoided

Reply:
This was intentional to avoid duplicating the same switch statement four times.